### PR TITLE
Pratiksha work

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

**Review by MKadeer17 — APPROVED** [review]

**Review by poonam-khode — COMMENTED** [review]
make yourself assignee and sequence of the questions is not correct

File: Pratiksha/Javascript/Assignment7/Q5.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <title>Insetion_Sort</title>
+  </head>
+  <body>
+    <script>
+      //Defining array
+      var stdCount = parseInt(prompt("how many student u want to enter?"));
+      var input_arr = [];
+      //Input
+      for (var i = 0; i < stdCount; i++) {
+        for (var mark = 0; mark < 5; mark++) {
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment7/Q5.html:16]: mark for loop is not required

File: Pratiksha/Javascript/Assignment9/Q3.html [hunk 2]
```diff
@@ -4,25 +4,24 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>doble_letter</title>
     <title>Q3</title>
   </head>
   <body>
     <script>
       var str = prompt("ENter String");
       console.log(str);
       console.log(doubleletter("tomorrow"));
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment9/Q3.html:14]: no call given to function doubleletter - doubleletter(str)

File: Pratiksha/Javascript/Assignment9/Q4.html [hunk 3]
```diff
@@ -4,19 +4,17 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Q4</title>
+    <title>String_palindrome</title>
   </head>
   <body>
     <script>
       const string = prompt("Enter a string: ");
       const value = checkPalindrome(string);
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment9/Q4.html:12]: the value should be displayed
 alert(value);

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**